crates/bevy_a11y/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+use std::num::NonZeroU128;
```

Comment:
Ideally we should have `#[forbid(unsafe_code)]` and `#[warn(missing_docs)]` on every possible first-party crate, this new crate is no exception.